### DISCUSSION TITLE
Commodore Amiga 500, 600 and 1200

### DISCUSSION THREAD

**Discussion by chapas** [darthcloud/BlueRetro#973] [General]
Hi,
I remember back in the day I could use a megadrive/genesis controller on my Amiga 500, can I use blueretro + genesis/megadrive adapter in my Amiga 500 today or do I need some adapter/modification for it to work. Can someone recommend if this is doable?
Can I use blueretro with a mouse on the Amiga as well?

Regards.

**Comment by darthcloud** [comment]
Yes but use the Parallel 1P or Parallel 2P firmware and wiring instruction (likely you need Atari type)

The Genesis FW will only work with Genesis games.

**Reply by chapas** [reply]
@darthcloud,
thanks for the reply, is there a schematic that I can follow?
Will the Amiga mouse be able to work as well with the Parallel firmware?

**Reply by darthcloud** [reply]
Mouse are not supported.

https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-Cables-Build-Instructions#parallel-2p-5v-6-buttons-each-adapter-cable

**Reply by chapas** [reply]
Adding this specific mouse is something that is planned? Can I request it?

**Reply by chapas** [reply]
@darthcloud, feasible?

**Reply by gbraad** [reply]
@darthcloud would rewiring the MD plug, like pin5 (pin7) be enough to make this work properly? This is what I have done before, with things like the 8bitdo retro receiver.

**Reply by Sowden** [reply]
I have asked about mouse support for the Atari ST.  It has most of the same protacalls, and I have been told that mouse support is not being planned to be implemented 😕

**Comment by chapas** [comment]
I believe I need the Open Drain PCB (OD v1.0) , but image bellow the provided link shows the LV 1.1, can you clarify?

**Reply by chapas** [reply]
@darthcloud, please clarify, thanks!
